internal/command/command.go

```diff
@@ -190,36 +193,3 @@ func generateSearchPattern(flags []string, delimiters []rune, padPattern bool) s
 	// example: (?<=[\"'\`])(flag1|flag2|flag3)(?=[\"'\`])"
 	return lookBehind + "(" + flagRegex + ")" + lookAhead
 }
-
-func normalizeAndValidatePath(path string) (string, error) {
```

Comment:
Factored this out into a new package since we're using it in multiple locations now